src/robot_dart/control/pd_control.cpp

```diff
@@ -26,9 +27,21 @@ namespace robot_dart {
             Eigen::VectorXd q = robot->positions(_controllable_dofs);
             Eigen::VectorXd dq = robot->velocities(_controllable_dofs);
 
+            Eigen::VectorXd error = Eigen::VectorXd::Zero(_control_dof);
+            for (int i = 0; i < _control_dof; ++i) {
+                if (robot->dof(_controllable_dofs[i])->getJoint()->getType() == dart::dynamics::RevoluteJoint::getStaticType()
+                    and _angular_errors) {
```

Comment:
`and` is python I guess. This should never be able to compile in C++. Please merge master into this branch so that we get the CI also..

Reply:
Oh wait. No need to merge. The CI is already running. But for some unknown reason this compiled! I'll investigate this..

Reply:
Wow. This is valid in C++: https://stackoverflow.com/questions/44381428/and-keyword-in-c

Still I prefer `&&`. Can you please adapt this?

Reply:
done